src/ontology/mondo-edit.obo

```diff
@@ -669,11 +669,15 @@ relationship: has_characteristic MONDO:0021152 {source="OMIMPS:603933"} ! inheri
 
 [Term]
 id: MONDO:0000066
-name: mitochondrial complex deficiency
+name: mitochondrial respiratory chain complex deficiency
+def: "A mitochondrial energy metabolism disorder where respiratory complex (I–V) is dysfunctional, typically due to mutations in genes encoding that specific complex’s proteins or assembly factors." [PMID:26741492]
 subset: gard_rare {source="MONDO:GARD"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
+synonym: "mitochondrial complex deficiency" EXACT []
 is_a: MONDO:0004069 {source="https://orcid.org/0000-0001-5208-3432"} ! inborn mitochondrial metabolism disorder
+is_a: MONDO:0016387 {source="Orphanet:254846"} ! mitochondrial oxidative phosphorylation disorder
```

Comment:
If Orphanet:254846 is the source, then it should also be a x-ref for this term.
Would Orphanet:254846 actually be equivalent to this Mondo term?